docs/tutorial/00-using-this-tutorial.md

```diff
@@ -2,7 +2,7 @@
 
 Each of the examples in this tutorial can be run on PyScript.com, a web environment that lets you write, test, and share PyScript code. Alternatively, you can clone the [PuePy git repo](https://github.com/kkinder/puepy) and run a live web server with each example included.
 
-The PyScript.com environment uses the PuePy `.whl` file, as [downloadable from PyPi](https://github.com/kkinder/puepy), while the examples in the git repository are served with PuePy directly from source files on disk.
+The PyScript.com environment uses the PuePy `.whl` file, as [downloadable from PiWheels](https://www.piwheels.org/project/puepy/), while the examples in the git repository are served with PuePy directly from source files on disk.
```

Comment:
I'm not familiar with piwheels.org. What is the advantage it has over PyPi: https://pypi.org/project/puepy/#files?

Reply:
Frankly I am also not familiar with pywheels.

Actually, I meant that the URL you had mentioned 

```
[downloadable from PyPi]
```

Was pointing to

```
(https://github.com/kkinder/puepy)
```

I couldn't find the .whl file from the github.com link. 
The link you meant was https://pypi.org/project/puepy/#files right?

Reply:
Ah, I see. Yes, that link should be fixed. Good catch.